utils/ALA_dialogues.cfg

```diff
@@ -437,7 +437,7 @@
 #define 03_FINAL_DIALOGUE
     [message]
         speaker=Glauco
-        message= _ "The station is free, and the river, too! Good work, my soldiers!"
+        message= _ "The station is free and so is the river! Good work, my soldiers!"
```

Comment:
I don't really see why this is necessary?

Reply:
Less stuttering way?

Reply:
yeah I don't think that's really necessary; please just leave this part as-is

Reply:
reverted in [b06d6ed](https://github.com/cooljeanius/A_Little_Adventure/pull/23/commits/b06d6edce5f6713420e42cbf7354dc358b29095b)